scripts/e2e-test-op-commands.py

```diff
@@ -0,0 +1,137 @@
+import json
+import os
+from pathlib import Path
+from typing import Any
+
+import click
+import structlog
+from eth_account.account import LocalAccount
+from eth_utils import to_checksum_address
+from hexbytes import HexBytes
+from web3 import Web3
+
+from beamer.contracts import contracts_for_web3, make_contracts, prepare_deployment_infos
+from beamer.typing import URL, ChainId
+from beamer.util import account_from_keyfile, make_web3, transact
+from scripts.call_contracts import pass_args
+
+logger = structlog.get_logger(__name__)
```

Comment:
We are importing `structlog`, initializing the logger and then... just use regular `print` statement.
Let's then remove `structlog` completely from this script.